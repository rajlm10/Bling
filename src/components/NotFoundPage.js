import React from 'react'
import {Link} from 'react-router-dom'

const NotFoundPage=()=>{
    return (
        <div>
            <p>This is a  404! <Link to="/">Go Home </Link></p>
        </div>
    )
}

export default NotFoundPage