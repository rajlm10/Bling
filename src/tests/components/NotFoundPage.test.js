import React from 'react'
import {shallow} from 'enzyme'
import NotFoundPage from '../../components/NotFoundPage'

test('render not found page',()=>{
    const wrapper=shallow(<NotFoundPage/>)
    expect(wrapper).toMatchSnapshot()

})