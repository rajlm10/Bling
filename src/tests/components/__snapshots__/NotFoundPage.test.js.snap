// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render not found page 1`] = `
<div>
  <p>
    This is a  404! 
    <[object Object]
      to="/"
    >
      Go Home 
    </[object Object]>
  </p>
</div>
`;
